### DISCUSSION TITLE
Signup in Auth0 (urgent)

### DISCUSSION THREAD

**Discussion by KetulCodiste** [Web3Auth/Web3Auth#604] [Help]
I am using web3Auth + Auth0
When my user want to loginin, web3Auth provides the connectTo() method which will redirect us to the auth0 login page.
But if the new user comes in, he needs to signup first. As of now i am using connectTo(), same flow as the login. Here the user will have the option of signup.
But this will confuse the user. Is there any method in web3Auth which will directly redirect  the new user to signup page of auth0 instead of the loginin page.

**Comment by rinzler99** [comment]
Hey, if you can check in your app wether the user is new or existing based on that you can redirect the user to signup or login screen.

Please refer below link to redirect user to signup screen of auth0.
https://community.auth0.com/t/signup-how-to-redirect-to-signup-page/39306
